packages/node-firebird-native-api/package.json

```diff
@@ -5,16 +5,24 @@
   "gypfile": true,
   "main": "dist/lib/index.js",
   "scripts": {
+    "install": "node-pre-gyp install --fallback-to-build",
+    "prepublishOnly": "yarn run build && yarn run gyp:clean && yarn run gyp:configure && yarn run gyp:build && yarn test",
+    "test": "jest",
     "build": "yarn run clean && yarn run lint && tsc",
     "build:w": "yarn run clean && tsc -w",
     "clean": "rimraf dist",
-    "lint": "tslint --project .",
     "generate": "node dist/generate-files",
-    "gyp:clean": "rimraf build",
-    "gyp:build": "node-gyp build",
-    "gyp:configure": "node-gyp configure",
-    "prepublishOnly": "yarn run build && yarn run gyp:clean && yarn run gyp:configure && yarn run gyp:build && yarn test",
-    "test": "jest"
+    "gyp:build": "node-pre-gyp build",
+    "gyp:clean": "rimraf build build-pre-gyp",
+    "gyp:configure": "node-pre-gyp configure",
+    "lint": "tslint --project .",
+    "pre-gyp:prebuild": "yarn run gyp:clean && node-pre-gyp configure",
+    "pre-gyp:publish": "yarn run pre-gyp:publish-darwin && yarn run pre-gyp:publish-linux && yarn run pre-gyp:publish-linux-ia32 && yarn run pre-gyp:publish-win32 && yarn run pre-gyp:publish-win32-ia32",
```

Comment:
@punkusha how can `pre-gyp:publish` work? How it will build multiple architectures in a single run?

Reply:
`node-pre-gyp` doesn't support parallel build. so, we can publish only 1 binary at once.
`node-pre-gyp-github publish --release` publish binary to github release, if github release is already created then update it.

run `pre-gyp:publish` to publish all the binaries (result: https://github.com/punkusha/node-firebird-drivers/releases/tag/node-firebird-native-api%400.1.1)